cookbooks/cdo-apps/libraries/cdo_apps.rb

```diff
@@ -106,6 +106,9 @@ def setup_app(app_name)
       # Reload when application is rebuilt.
       subscribes :reload, 'execute[build-cdo]', :delayed
 
+      # Reload when global config is updated to pick up changes.
+      subscribes :reload, 'template[globals]', :delayed
```

Comment:
The notification from a `template` resource is triggered whenever the contents of its _rendered_ template are changed, which in this case would mean whenever `globals.yml` contains any new content. Our application server needs to restart in order to pick up config changes, so this is the behavior we actually want.

I needed to add this in order to trigger an application-server reload when enabling/disabling proxy integration, which changes the globals.yml config output for the mysql endpoint the application connects to.

Keep in mind that in our setup we only run chef-client manually during CI builds, so that's the only time this will ever be triggered in our normal operation.